wp-content/themes/core/components/quote/Quote_Controller.php

```diff
@@ -7,61 +7,45 @@
 use Tribe\Project\Templates\Models\Image;
 use Tribe\Project\Theme\Config\Image_Sizes;
 
-class Controller extends Abstract_Controller {
-	/**
-	 * @var string[]
-	 */
-	private $classes;
-	/**
-	 * @var string[]
-	 */
-	private $attrs;
-	/**
-	 * @var string
-	 */
-	public $quote_text;
-	/**
-	 * @var string
-	 */
-	public $cite_name;
-	/**
-	 * @var string
-	 */
-	public $cite_title;
-	/**
-	 * @var string
-	 */
-	private $cite_image;
+class Quote_Controller extends Abstract_Controller {
+	public const CLASSES    = 'classes';
+	public const ATTRS      = 'attrs';
+	public const QUOTE_TEXT = 'quote_text';
+	public const CITE_NAME  = 'cite_name';
+	public const CITE_TITLE = 'cite_title';
+	public const CITE_IMAGE = 'cite_image';
 
-	public function __construct( array $args = [] ) {
-		$args = wp_parse_args( $args, $this->defaults() );
-
-		foreach ( $this->required() as $key => $value ) {
-			$args[$key] = array_merge( $args[$key], $value );
-		}
+	private array $classes;
+	private array $attrs;
+	public string $quote_text;
```

Comment:
Prob should update these to private and add getters for them, yes?

Reply:
Yep, I was planning to revisit